docs/Model/WebAuthnType.md

```diff
@@ -0,0 +1,8 @@
+# # WebAuthnType
+
+## Properties
+
+Name | Type | Description | Notes
+------------ | ------------- | ------------- | -------------
+
```

Comment:
@danilo-kaltner or @bertrmz the SDK generator has created this type with no properties. Is this correct?

Reply:
That's correct; the type is an enum of strings, so the strings themselves have no properties.

I'm surprised the valid enum values aren't showing up here though; they're set in [the spec](https://app.swaggerhub.com/apis/passage/passage-mgmt-sdk-api/1#/WebAuthnType) like:
<img width="381" alt="Screenshot 2023-10-31 at 4 13 01 PM" src="https://github.com/passageidentity/passage-php/assets/13988480/c9202738-0b6e-4d32-8a68-3b5c6b25994b">

Reply:
Hmm that is a little concerning. The enum should have been created like this:
https://github.com/passageidentity/passage-php/blob/aa81aeac8b66e415b941dc0609e0f6b76d0f1d3b/generated/lib/Model/MagicLinkType.php#L54

Reply:
Upon digging, it looks like none of our enum types are properly documented. Here are the [MagicLinkTypes](https://github.com/passageidentity/passage-php/blob/main/docs/Model/MagicLinkType.md). I will approve this PR but then check if there is a setting we can change to make this generate properly.